__tests__/app.test.js

```diff
@@ -156,4 +156,57 @@ describe('App', () => {
         })
     });
   });
+  describe('POST /api/articles/:article_id/comments', () => {
+    test("201: returns the new comment object", () => {
+      return request(app)
```

Comment:
It would be nice to have another test where you do a .get to the comments for the new comment once it's been added to prove it's gone in.